doc/man1/build.info

```diff
@@ -51,10 +52,11 @@ DEPEND[openssl-dsa.pod]=../perlvars.pm
 GENERATE[openssl-dsa.pod]=openssl-dsa.pod.in
 DEPEND[openssl-dsaparam.pod]=../perlvars.pm
 GENERATE[openssl-dsaparam.pod]=openssl-dsaparam.pod.in
+DEPEND[openssl-ec.pod]=../perlvars.pm
+GENERATE[openssl-ec.pod]=openssl-ec.pod.in
 DEPEND[openssl-ecparam.pod]=../perlvars.pm
 GENERATE[openssl-ecparam.pod]=openssl-ecparam.pod.in
 DEPEND[openssl-ec.pod]=../perlvars.pm
-GENERATE[openssl-ec.pod]=openssl-ec.pod.in
```

Comment:
Why is this being re-ordered? We now have a duplicate `DEPEND` line for `openssl-ec`

Reply:
Rebase error.  Sorting, "ec." comes before "ecparam" :)  Fixed.